webpack/common.config.js

```diff
@@ -2,7 +2,7 @@ const webpack = require('webpack')
 const { merge } = require('webpack-merge')
 const path = require('path')
 const MiniCssExtractPlugin = require('mini-css-extract-plugin')
-const CopyWebpackPlugin = require('copy-webpack-plugin')
+const CopyWebpackPlugin = require('copy-webpack-plugin') // eslint-disable-line
```

Comment:
Temporary fix. Is CopyWebpackPlugin used at all?

Reply:
It could be that I just copied it from another config, but I think we will need it in the future to copy assets like images.

Reply:
Alright.
So do we keep the import but silence the linter error or remove it and reintroduce it once the images copying actually happens?
Its just one line.

Reply:
Well, lets just delete the line.